tardis/plasma/properties/atomic.py

```diff
@@ -39,7 +39,7 @@ def _set_index(self, levels):
         return (levels.index, levels['energy'], levels['metastable'],
             levels['g'])
 
-class Lines(BaseAtomicDataProperty):
```

Comment:
I'm back soon. We'll talk about a fix then.
On Mon, Jan 25, 2016 at 13:19 aoifeboyle <notifications@github.com> wrote:

> In tardis/plasma/properties/atomic.py
> <https://github.com/tardis-sn/tardis/pull/466#discussion_r50675519>:
>
> > @@ -39,7 +39,7 @@ def _set_index(self, levels):
> >          return (levels.index, levels['energy'], levels['metastable'],
> >              levels['g'])
> >
> > -class Lines(BaseAtomicDataProperty):
>
> @wkerzendorf <https://github.com/wkerzendorf> I don't know how to fix
> this while leaving it as a BaseAtomicDataProperty. Could you recommend an
> alternative way of changing set_index so that it doesn't re-order the
> data?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tardis-sn/tardis/pull/466/files#r50675519>.
>
-- 
Dr. Wolfgang E Kerzendorf
ESO Fellow
European Southern Observatory
Karl-Schwarzschild Str. 2, 85748
Garching bei Muenchen, Germany